### ISSUE TITLE
reek not picking up local Attribute exclusion 

### ISSUE THREAD

**Issue by jcarapet** [troessner/reek#751]
Hi there. 

Reek is blowing up on a writeable attribute (attr_accessor), and I was hoping to exclude only one version of it vs. all. I have followed same style as others on single exclusion  (# :reek:IrresponsibleModule), but reek isn't picking it up . putting exclusion in .reek file works however. 
Working
``` Attribute:
  enabled: false
  exclude: []
```
Non working locally 
``` 
# exclusion for reek is failing
# :reek:attribute
attr_accessor :state
```

Am I doing something incorrectly, or is this a bug?

**Comment by mvz** [comment]
Have you tried writing attribute with a capital 'A'?

**Comment by jcarapet** [comment]
yes, and tried a space after comment, putting at top of file vs just above offense, and :reek:Attribute_writer just for fun.

**Comment by mvz** [comment]
Ok, then it's actually a bug :smile:.

**Comment by troessner** [comment]
Confirmed, I can easily reproduce this.

**Comment by mvz** [comment]
Fix in #752.

**Comment by troessner** [comment]
Fixed in master and released as soon as #753 is merged (which probably will happen in the next couple of hours) -> closing this one.
